property.php

```diff
@@ -14,18 +14,15 @@
     <link rel="stylesheet" href="./css/style.css" type="text/css">
 </head>
 <body>
-<div class="container">
-    <nav class=“navbar navbar-light bg-light mb-4”>
-        <div class=“container”>
-            <a class=“navbar-brand” href=“#”>
-                <img src=“./img/logo.png” height=“40"
-                     class=“d-inline-block align-top” alt=“”>
-            </a>
-            <h2>Armadillo Estates</h2>
-            <div></div><!-- this empty div is required for the positioning of the h2 above-->
-        </div>
-    </nav>
-</div>
+<nav class="navbar navbar-light bg-light mb-4">
+    <div class="container">
+        <a class="navbar-brand" href="index.php">
+            <img src="./img/logo.png" class="d-inline-block align-top logoimg" alt="">
+        </a>
+        <h2>Armadillo Estates</h2>
+        <div class="stabilizerDiv"></div><!-- this empty div is required for the positioning of the h2 above-->
```

Comment:
any reasons there's a class to this? You're applying height and width to it in the css but why?